CHANGELOG.md

```diff
@@ -1,9 +1,12 @@
 # Changelog
 
 ### 2.3.8 (TBD)
```

Comment:
This is maybe changed in other PRs but this should be 2.4.0, but also something we can fix before building an rc so not a blocker.

Reply:
Yeah, I think it says 2.3.8 on the current release/v2 branch too, so let's fix that later on.